### PR TITLE
Added social and slides to cracking the coding interview

### PR DESCRIPTION
**Pull request by EricGoat**
@zeroBitOfDiff New update sensei

### CODE REVIEW

File: _data/config.yml [hunk 1]
```diff
@@ -14,19 +14,19 @@
   workshopDisplayLimit: 4
   events:
     -      
-      name: "Cracking the Coding Interview"
-      date: 2/24/2018 
-      time: 12:00pm
-      url: "https://www.facebook.com/events/1737579479682995/"
+      name: "Tech Knights Social"
+      date: 4/1/18
```
Comment (maxcell) [inline _data/config.yml]: Do you mean March 1st?

Comment (maxcell) [reply _data/config.yml]: @EricGoat

Comment (EricGoat) [reply _data/config.yml]: That's what it is, its in mm/dd/yy @maxcell

Comment (maxcell) [reply _data/config.yml]: It current written as '4/1/18'

Comment (maxcell) [reply _data/config.yml]: (March is month 3) :heart:

Comment (EricGoat) [reply _data/config.yml]: Oh wow dumb me that's what I meant

Comment (maxcell) [reply _data/config.yml]: No worries @EricGoat! That's why I always check it out 💖

**Comment by EricGoat** [comment]
@maxcell updated with the correct month

**Pull request merged**